.rubocop.yml

```diff
@@ -0,0 +1,9 @@
+inherit_gem:
+  rubocop-rails-omakase: rubocop.yml
+
+Layout/SpaceInsideArrayLiteralBrackets:
+  Enabled: false
```

Comment:
In addition to not liking this style myself, having it enabled requires changing almost all array literals in the codebase (~500 more offences). So I decided to not take this one in.